src/shared/cloudformation/activation.ts

```diff
@@ -17,8 +19,18 @@ export const TEMPLATE_FILE_GLOB_PATTERN = '**/template.{yaml,yml}'
  * @param extensionContext VS Code extension context
  */
 export async function activate(extensionContext: vscode.ExtensionContext): Promise<void> {
-    const registry = CloudFormationTemplateRegistry.getRegistry()
-    const manager = new CloudFormationTemplateRegistryManager(registry)
-    await manager.addTemplateGlob(TEMPLATE_FILE_GLOB_PATTERN)
-    extensionContext.subscriptions.push(manager)
+    try {
+        const registry = CloudFormationTemplateRegistry.getRegistry()
+        const manager = new CloudFormationTemplateRegistryManager(registry)
+        await manager.addTemplateGlob(TEMPLATE_FILE_GLOB_PATTERN)
+        extensionContext.subscriptions.push(manager)
+    } catch (e) {
+        getLogger().error(
+            localize(
+                'AWS.codelens.failToInitialize',
+                "Unable to activate the template registry, SAM CodeLense's will not work"
```

Comment:
Change to "Issue activating template registry. CodeLenses will not appear on SAM template files."

Reply:
"Issue activating" sounds a bit awkward, consider:

> Failed to activate template registry. CodeLenses will not appear on SAM template files.

"Failed to" or "Cannot" are clear and unambiguous in pretty much every case.

Reply:
Rather than let this silently fail, is there merit in showing an error notification?

Reply:
This doesn't put it in the log? what?

Reply:
Unsure what you're saying? This adds it to the AWS Toolkit log + backing file, but does not pop up any other notifications (the user would have to go to the log and there wouldn't be any prompting to do so, unlike Jetbrains which has an error alert).

I do second @awschristou 's comment, since the rest of the toolkit will work, it's worth letting the user know what isn't working.

Reply:
it's an error message now

Reply:
Can you swap the localized message to @justinmk3 's suggestion?

Reply:
Changed